src/identity/auth.ts

```diff
@@ -51,11 +51,12 @@ export class Auth {
 		let strategy: oauth2.Strategy
 		strategy = new oauth2.Strategy(
 			{
-				authorizationURL: `${this.config.authenticationServiceUrl}/oauth/authorize`,
+				authorizationURL: this.config.authorizationPath,
```

Comment:
This should be ``authorizationURL: `${this.config.authenticationServiceUrl}${this.config.authorizationPath}` `` I think

Reply:
Comment applied.